lib/lightstep/global_tracer.rb

```diff
@@ -15,7 +15,11 @@ def initialize
     # Configure the GlobalTracer
     # See {LightStep::Tracer#initialize}
     def configure(**options)
-      raise ConfigurationError, 'Already configured' if configured
+      if configured
+        LightStep.logger.warn "LIGHTSTEP WARNING: Already configured. Stack trace:\n\t#{caller.join("\n\t")}"
```

Comment:
I can remove if you think that's better... my hope was to help the user understand where the double-call came from. I'll make the stack trace an `info` rather than a `warn`.